cinema/models.py

```diff
@@ -35,12 +41,21 @@ def full_name(self):
         return f"{self.first_name} {self.last_name}"
 
 
+def movie_image_file_path(instance: Movie, filename: str):
+    _, extension = os.path.splitext(filename)
+
+    filename = f"{slugify(instance.title)}-{uuid.uuid4()}.{extension}"
+
+    return os.path.join("uploads/movies/", filename)
```

Comment:
You should join `uploads` and `movies` parts as well to not stick to a specific filesystem. Consider this in the future.